Confus/Confus/Flag.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+namespace irr 
+{
+    class IrrlichtDevice;
+    namespace scene {
+        class IAnimatedMeshSceneNode;
+    }
+}
+
+namespace Confus 
+{
+	class Player;
```

Comment:
You can ommit the 'namespace Confus' and just place this in the namespace you are declaring below here